fast64_internal/sm64/sm64_objects.py

```diff
@@ -758,26 +755,28 @@ def handleRefreshDiffModelIDs(modelID):
 
 
 def handleRefreshDiffSpecials(preset):
+    refresh_version = bpy.context.scene.fast64.sm64.refresh_version
     if (
-        bpy.context.scene.refreshVer == "Refresh 8"
-        or bpy.context.scene.refreshVer == "Refresh 7"
-        or bpy.context.scene.refreshVer == "Refresh 6"
-        or bpy.context.scene.refreshVer == "Refresh 5"
-        or bpy.context.scene.refreshVer == "Refresh 4"
-        or bpy.context.scene.refreshVer == "Refresh 3"
+        refresh_version == "Refresh 8"
+        or refresh_version == "Refresh 7"
+        or refresh_version == "Refresh 6"
+        or refresh_version == "Refresh 5"
+        or refresh_version == "Refresh 4"
+        or refresh_version == "Refresh 3"
```

Comment:
Usually when I do this I use [, but this will be a set right? Is that recommended?

Reply:
yeah sets are faster/scale better since the lookup is constant cost
it's what I typically see in good python code but tbh nothing here is performance critical / we don't care / it doesn't change much, so a list, tuple or set is whatever lol